net.blix.BlueMail.yaml

```diff
@@ -0,0 +1,73 @@
+app-id: net.blix.BlueMail
+default-branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: bluemail
+tags: [proprietary]
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=xdg-config/gtk-3.0
```

Comment:
Exposing host's GTK3 config sounds like a can of worms. What is it needed for?

Reply:
I was having some issues with Electron and this was leftover from testing. Sorry about that